src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -208,6 +208,14 @@ def load_arguments(self, _):
     with self.argument_context('webapp config connection-string') as c:
         c.argument('connection_string_type', options_list=['--connection-string-type', '-t'], help='connection string type', arg_type=get_enum_type(ConnectionStringType))
 
+    with self.argument_context('webapp config azure-storage-accounts') as c:
+        c.argument('id', options_list=['--id', '-i'], help='custom identifier')
+        c.argument('storage_type', options_list=['--storage-type', '-t'], help='storage type', arg_type=get_enum_type(AzureStorageType))
+        c.argument('account_name', options_list=['--acount-name', '-a'], help='storage account name')
+        c.argument('share_name', options_list=['--share-name', '-sn'], help='share name (Azure Files) or container name (Azure Blob Storage)')
```

Comment:
Short names with 2+ characters are not commended, please add one more dash, or just remove it.
Cc: @tjprescott